### ISSUE TITLE
Generate a Feature when registration form has GPS cordinates

### ISSUE THREAD

**Issue by githengi** [opensrp/opensrp-client-chw-emergency-transport#9]
- [x] Generate a Feature when registration form has GPS coordinates.

- [x] The domain used to save should be https://github.com/OpenSRP/opensrp-client-core/blob/master/opensrp-app/src/main/java/org/smartregister/domain/Location.java

- [x] The feature should be saved on the structures table.

There are 2 options for handling sync - Option 1 will be used
1.  Save feature during client processing; extract feature from event. This will remove need to sync locations. This would assume structures cannot be loaded server side but are generated on the client

~2.  (NOT USED) Save structures during post save actions and sync structure to the server. Clients will be syncing structures downstream also for structures~

This will improve operations for querying data to display on the map

- [x] Set the properties type to `Residential Structure`. This will be use in #2 to color this feature on the map

**Comment by githengi** [comment]
LOE 3 days for 1 FTE

**Comment by msschroeder** [comment]
@githengi Is this up to date based on what was discussed in the last meeting on Friday? It looks okay, but I am not sure about the sync - what option we decided to go with. I think number 1?

**Comment by githengi** [comment]
Yes number 1 was chosen where features would be generated at runtime during client processing. Where GPS coordinates are saved as the features in the structures table
